test/langtools/tools/javac/patterns/LocalVariableTable.java

```diff
@@ -26,8 +26,8 @@
  * @bug 8231827
  * @summary Ensure the LV table entries are generated for bindings
  * @modules jdk.jdeps/com.sun.tools.classfile
- * @compile -g --enable-preview -source ${jdk.version} LocalVariableTable.java
- * @run main/othervm --enable-preview LocalVariableTable
+ * @compile -g LocalVariableTable.java
```

Comment:
I believe all tests are always compiled with `-g` option set, not related to your patch but we could probably remove that line as superfluous

Reply:
I believe plain jtreg invocations may not be always setting "-g". So probably better to be explicitly clear we need -g, as the test itself requires the debugging info/LocalVariableTable?

Reply:
sure I understand that the `-g` option is necessary. I was just wondering if `-g` is not passed always by jtreg. If the option is passed by default then it is superfluous, although just a comment, I understand that it is safer to pass it

Reply:
I tried with a plain jtreg invocation, and it didn't pass -g (and the test failed). But it depends on how jtreg is run, of course - some runs may have -g set.